resources/developer-mode.mjs

```diff
@@ -246,6 +274,16 @@ function updateURL() {
     else
         url.searchParams.delete("useWarmupSuite");
 
+    if (params.warmupBeforeSync !== defaultParams.warmupBeforeSync)
+        url.searchParams.set("warmupBeforeSync", params.warmupBeforeSync);
+    else
+        url.searchParams.delete("warmupBeforeSync");
+
+    if (params.waitBeforeSync !== defaultParams.waitBeforeSync)
+        url.searchParams.set("waitBeforeSync", params.waitBeforeSync);
+    else
+        url.searchParams.delete("waitBeforeSync");
+
```

Comment:
That's 4 times the same code, what do you think of replacing these lines with something such as:
```js
for (const paramKey of [
  'iterationCount',
  'useWarmupSuite',
  'warmupBeforeSync',
  'waitBeforeSync',
]) {
  if (params[paramKey] !== defaultParams[paramKey]) {
    url.searchParams.set(paramKey, params[paramKey]);
  } else {
    url.searchParams.delete(paramKey);
  }
}
```
This would also avoid typos when repeating the param keys.